app/register/page.tsx

```diff
@@ -296,14 +343,14 @@ const Register: React.FC = () => {
         </Grid>
       </form>
       {errorMessages.length > 0 && (
-              <ul>
-                {errorMessages.map((errorMessage, index) => (
```

Comment:
Don't use pure HTML elements. Please use List and ListItem from MUI

Reply:
This part is a backend part. Is it okay to Update?